tests/manage/mcg/test_s3_prefix_list.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.mcg import MCG
+from ocs_ci.ocs.bucket_utils import (
+    write_random_objects_in_pod,
+    copy_objects,
+    list_objects_from_bucket,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+    tier2,
+    skipif_ocs_version,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@bugzilla("2068110")
+@pytest.mark.polarion_id("OCS-3925")
+@tier2
+@skipif_ocs_version("<4.11")
+class TestS3PrefixList:
+
+    """
+    Test S3 prefix list operations
+    """
+
+    def test_s3_prefix_list(
+        self,
+        awscli_pod_session,
+        bucket_factory,
+        test_directory_setup,
+    ):
+        """
+        Test s3 prefix uploads and list operations
+        """
+
+        bucket = bucket_factory()[0]
+        bucket_name = bucket.name
+        file_dir = test_directory_setup.origin_dir
+        s3_obj = MCG(bucket_name)
+        prefix = "mrbts"
```

Comment:
Does this prefix have a meaning? If it does - please document it. If it doesn't - please use a more descriptive one.